src/xdocs/config_regexp.xml

```diff
@@ -570,6 +577,33 @@ public static final int A_SETPOINT = 1;
     value=&quot;System\.(out)|(err)\.print(ln)?\(&quot;/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To configure the check to find match text that spans multiple lines:
+        </p>
+        <source>
+&lt;module name=&quot;RegexpMultiline&quot;&gt;
+  &lt;property name=&quot;matchAcrossLines&quot; value=&quot;true&quot;/&gt;
+  &lt;property name=&quot;format&quot; value=&quot;start.*end&quot;/&gt;
+&lt;/module&gt;
+        </source>
+        <p>
+          Example of violation from the above config:
+        </p>
+        <source>
+start line # only 1 violation
```

Comment:
can you share real example what for such option might be used ?

in issue, example is artificial too, It is better to show user a case where it might be used.

Reply:
real example was to search for specific lines of code (bad version) and convert it to a new version. I don't remember what my real case was from the time, but I'll try to come up with something.

Reply:
I can merge, if you create outbound issue to update doc with real example.
in such case this update will be in 8.25 release, so you will be able to run it over your code.

Reply:
Done and pushed.
I used the previous example and expanded it to `System.out` that spans more than 1 line.